WorkingAtBabylon.md

```diff
@@ -0,0 +1,89 @@
+
+### Does your calendar look like a losing game of tetris? Do you have large continuous uninterrupted hours?
+It depends a lot on the squad. For Enrolment and Integrity, for example, most meetings are on the first Monday and Tuesday of the sprint. For Triage and Health Management this is not the case and are more spread out.
+
+### Are there any private spaces in your office?
+There are very few at the moment.
+
+### Is there a gym/field/court close to your office for physical activity?
+There is a commercial gym close by, a park, and a barebones gym in the office. There’s also free yoga twice a week.
+
+### Do people feel comfortable calling attention to systemic issues/heavily “band-aid”-ed systems?
+Yes, we have anonymous company level Q&A and peakon surveys. [this is my experience].
+
+### Are any of your engineering managers mobile engineers? Do mobile devs feel like they have to learn backend dev to "prove themselves" / feel like they have a seat at the table?
+iOS engineers have to know iOS, and that’s it.
+
+## Designers?
+### What’s the handoff process like?
+As we work in squads, collaboration with designers, product, and QA is very tight and happens every day. After a ticket has been discussed in discovery and planning, designs are provided on Zeplin (sketch).
+
+### Do you let devs get down and dirty with sketch files?
+Not usually, but developers are encouraged to report wrong alignments, spacing, and in general to partecipate in the design/UX discussion.
+
+### Do you focus on making your android and iOS apps look consistent?
+There is a drive for consistency with a design system, but platform-specific conventions are taken into account.
+
+### Example of the above: Do you have the “title” of a VC on the left in iOS/on the center in android for consistency? Do you have a modal View Controller presented with a back arrow on the left?
+We follow iOS standards, but it's up to every squad.
+
+## Q/A?
+### What’s the handoff-to-QA process like?
+Same answer as with designers. Collaboration is frequent as there is a QA per squad. A build is triggered from the main branch every night (with custom builds trigger-able at any time), and in the morning the QA starts testing against it.
+
+## Learning
+### Do you sponsor conference tickets?
+We have a set `training` budget that can be used at the person's discretion.
+
+### Books/online resources?
+We have a set `training` budget that can be used at the person's discretion.
+
+## Product
+### Do devs get access to support tickets?
+### How involved are devs in product decisions?
+There is usually a product person in each squad that leads this, but everyone is encouraged to contribute and/or raise concerns.
+
+### Do you consider the societal impact of the products you’re making?
+Yes. We wouldn't be here otherwise.
+
+### Does your marketing org ensure your marketing materials are inclusive (I like to call this the Revolut filter)?
```

Comment:
I don't get the point of this. Sounds like some accusation of a third company, the document should be free of such things imo.

Reply:
Will delete.